app/models/customization_script.rb

```diff
@@ -3,4 +3,5 @@ class CustomizationScript < ApplicationRecord
 
   acts_as_miq_taggable
   belongs_to :provisioning_manager
```

Comment:
Ah, this line will have to be removed with the next line added.  It might break some tests that have to be cleaned up.  But, the `provisioning_manager` and the `ext_mangement_system` are actually really the same thing (both pointing to `ext_management_systems` table).